### PR TITLE
Update about.md

### PR DESCRIPTION
**Pull request by Timanana**

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -168,9 +168,9 @@ The orgnization was created as a student run community service orgnization for C
 		<div class="direction-l">
 			<div class="flag-wrapper">
 				<span class="flag">A Non Profit Org</span>
-				<span class="time-wrapper"><span class="time">Apr, 2021 </span></span>
+				<span class="time-wrapper"><span class="time">Apr, 2021</span></span>
 			</div>
-			<div class="desc">ICC became a non profit orgnization in California.</div>
+			<div class="desc">ICC became a non profit orgnization in Irvine</div>
```
Comment (DeletedUser-9999) [inline about.md:173]: period?

File: about.md [hunk 2]
```diff
@@ -181,7 +181,7 @@ The orgnization was created as a student run community service orgnization for C
 				<span class="flag">PSVA Certifying Org</span>
 				<span class="time-wrapper"><span class="time">May, 2021</span></span>
 			</div>
-			<div class="desc">ICC was approved by Presidential Service Award as a Certifying Orgnization</div>
+			<div class="desc">ICC achieved the Presidential Volunteer Service Award and became a Certifying Organization</div>
```
Comment (DeletedUser-9999) [inline about.md:184]: I kinda like the old version better but that's just me

File: about.md [hunk 3]
```diff
@@ -236,7 +236,7 @@ Our members come from different schools in southern California.
 * Beacon Park School (K8)
 * Many More!
 
-If you are a 6th grade and above CS enthusiast who want to learn together with our team members,
+If you are a 6th grade and above CS enthusiast who wants to learn together with our team members,
 or if you are good at some areas that you want to share with our community by providing online courses.
 Come and join ICC by filling out [this form](https://forms.gle/Ky4S6YLkkursGyZn6).   
-Send an email to irvinecodingclub@gmail.com if you didn't get back from us.
+Send an email to <a href="mailto:irvinecodingclub@gmail.com">irvinecodingclub@gmail.com</a> if you didn't get back from us.
```
Comment (DeletedUser-9999) [inline about.md:242]: Great find with the unlinked email! Are we trying to use admin@irvinecoding.club or irvinecodingclub@gmail.com tho?

**Review by DeletedUser-9999 — CHANGES_REQUESTED** [review]
Okie here ya go

**Review by DeletedUser-9999 — APPROVED** [review]

File: about.md (hunk 1 above)
Comment (Timanana) [reply about.md]: none of the other things have periods

**Pull request merged**